playground/NodeEditor.js

```diff
@@ -297,7 +339,7 @@ export class NodeEditor extends THREE.EventDispatcher {
 
 		previewMenu.add( editorButton );
 
-		this.domElement.append( menu.dom );
+		document.body.appendChild(menu.dom);
```

Comment:
We need to move the menus to `domElement` in order to maintain the consistency of the hierarchy. It shoud be something like:
- editor class="editor"
  - splitview
  - f-menu
  - flow

Reply:
If you can follow the name style could be better intead of `div`, could remove some redundant classes name too. Something like this:

![image](https://github.com/mrdoob/three.js/assets/502810/08b881b7-4aab-4a43-8d04-73d235ece9e4)

`node-editor` is `domElement` in this case

- node-editor
  - f-preview ( instead of splitview )
  - flow
  - f-menu

Reply:
Updated.
![image](https://github.com/mrdoob/three.js/assets/21260178/134cbf23-2c7f-48bb-a276-2e690486f7af)
f-menu still lives in flow, because otherwise i would have to keep track of when the editor gets added in order to then add f-menu to the domElement parent.

is it fine like that?